src/views/PopularDevelopers.vue

```diff
@@ -1,12 +1,68 @@
 <template>
   <div>
     <h3 class="center-align">Popular Developers</h3>
-    <slot />
+    <p class="center">
+      Showing
+      <strong>{{ filteredDevelopers.length.toLocaleString() }}</strong> developers
```

Comment:
Actually, here we show the total of developers here.
```suggestion
      <strong>{{ developers.length.toLocaleString() }}</strong> developers
```

Reply:
I suppose that when virtual scroll get implemented `filteredDevelopers` will have all the developers that the first time, however I used `filteredDevelopers` cause when you use the filter I suppose would be better to count just the amount that really can be shown.

Reply:
The virtual scroll is a mimic of showing all the developers, from the user perspective we're showing everything is just that they haven't scroll down enough to see all the developers. You can see [here](https://developersdo.github.io/opensource/developers/popular) how this is implemented.

Reply:
Yes, but I for example someone filter to just those developers which name start with 'ger', we would just show those one, that was the reason why I used `filteredDevelopers` just to count those one that can be shown.